docs/source/api/user_login_oauth.rst

```diff
@@ -0,0 +1,73 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-user-login-oauth:
+
+********************
+``user/login/oauth``
+********************
+.. versionadded:: 1.4
+
+``POST``
+========
+
+Authentication of a user using an encrypted Json Web Token from an OAuth service. Traffic Ops will decode and validate the token, validate the key set domain, and send back a session cookie.
+
+:Auth. Required: No
+:Roles Required: None
+:Response Type:  ``undefined``
+
+Request Structure
+-----------------
+:t: Token
```

Comment:
this part now needs to be updated to reflect the new expected request format

Reply:
done

Reply:
You updated the example, but the structure explanation still has just `t`

Reply:
It looks like you just updated the example, but the structure explanation still just lists `t`